<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QDialog" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatDialog</string>
  </property>
  <widget class="QTextEdit" name="outputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="emojiButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../LoginWindow.qrc">
     <normaloff>:/LoginWindow/Resources/emoji.png</normaloff>:/LoginWindow/Resources/emoji.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../LoginWindow.qrc"/>
 </resources>
 <connections/>
</ui>
